apis/Google.Cloud.Datastore.V1/Google.Cloud.Datastore.V1/AggregationQueryResults.cs

```diff
@@ -0,0 +1,96 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at 
+//
+// https://www.apache.org/licenses/LICENSE-2.0 
+//
+// Unless required by applicable law or agreed to in writing, software 
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and 
+// limitations under the License.
+
+using Google.Protobuf.Collections;
+using Google.Protobuf.WellKnownTypes;
+using static Google.Cloud.Datastore.V1.QueryResultBatch.Types;
+
+namespace Google.Cloud.Datastore.V1;
+
+/// <summary>
+/// Represents the result of an <see cref="AggregationQuery"/>.
+/// </summary>
+public sealed class AggregationQueryResults
+{
+    /// <summary>
+    /// The result of the aggregation functions, ex: `COUNT(*) AS  total_entities`.
+    /// The key is the [alias][google.datastore.v1.AggregationQuery.Aggregation.alias] assigned
+    /// to the aggregation function on input and the size of this map equals the number
+    /// of aggregation functions in the query.
+    /// </summary>
+    private readonly MapField<string, Value> _aggregateProperties = new MapField<string, Value>();
```

Comment:
Now that it's easier to see how this is used, is there any reason it needs to be a `MapField` instead of just `Dictionary<string, Value>`?